### DISCUSSION TITLE
Welcome to Neumorphism Discussions! SWRevo by IndoSW

### DISCUSSION THREAD

**Discussion by SWRevo** [SWRevo/Neumorphism#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SketchDevo** [comment]
Where is the Sketchware Project

**Reply by SWRevo** [reply]
Inside Sketchware Folder

**Reply by Buddygupta** [reply]
Not found  in sketcware folder

**Comment by dhaniyell** [comment]
Password??

**Reply by Faisal4110** [reply]
Sir Zip Password ?

**Reply by dhaniyell** [reply]
Yup

**Reply by yusufquteb** [reply]
https://indosw.com/2020/12/21/neumorphism-implementation-android-studio-sketchware-pro/

**Reply by yusufquteb** [reply]
indosw2021*

**Comment by yusufquteb** [comment]
Can't open MTsk file

**Reply by Buddygupta** [reply]
What use of this file
